_config.yml

```diff
@@ -18,6 +18,11 @@ permalink: /blog/:title/
 paginate: 9,
 paginate_path: "/blog/page:num/"
 
+exclude: [
```

Comment:
when we implement webpack this goes away

Reply:
Yeah, at that point the bundle should include what we need.